src/app/inventory/dim-item-info.ts

```diff
@@ -140,6 +143,32 @@ export function cleanInfos(stores: DimStore[]): ThunkResult {
         const info = infos[item.id];
         if (info && (info.tag !== undefined || info.notes?.length)) {
           cleanupIds.delete(item.id);
+        } else {
+          // Double-check crafted items - we may have them under a different ID. If so,
+          // patch up the data by re-tagging them under the new ID.
+          if (item.craftedInfo?.craftedDate) {
+            const craftedInfo = infosByCraftedDate[item.craftedInfo.craftedDate];
```

Comment:
I was wondering if this should have some protection against "going back in time" when Bungie.net returns older info but that should be solved with the minted timestamp now anyway and would probably cause more problems than it solves.

Reply:
Yeah we shouldn't evaluate older profile responses anymore after my other PR lands.